src/core/ColorPicker/Color.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import { getWindow } from '../utils/common';
+import { CommonProps } from '../utils/props';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export type ColorProps = {
+  /**
+   * Color
+   */
+  color: string;
+  /**
+   * Handler for clicking a color
+   */
+  onColorClicked?: () => void;
+  /**
+   * Is color selected
+   */
+  isActive?: boolean;
+  /**
+   * User friendly color name
+   */
+  name?: string;
+  /**
+   * User Count Icon props.
```

Comment:
Not sure what is this comment about

Reply:
Fixed comment

Reply:
Yeah, but this has nothing to do with the tooltip.. this is the reference of the Color component, which can be used for whatever...
Check my other comment about ref, maybe that would make more sense.

Reply:
I removed this prop and use forwardRef instead